modules/openy_features/openy_location/modules/openy_loc_filter/openy_loc_filter.links.menu.yml

```diff
@@ -0,0 +1,5 @@
+openy_loc_filter.admin:
+  title: 'Open Y Location Filter settings'
+  parent: system.admin_config_content
```

Comment:
the menu item should be located under the Open Y menu. 
Open Y -> Settings -> Open Y Location Filter settings